src/libraries/System.Runtime/tests/System/Reflection/NullabilityInfoContextTests.cs

```diff
@@ -1348,4 +1409,14 @@ public DerivesFromTupleOfNestedGenerics(List<string[]> item1, Dictionary<object[
         {
         }
     }
+
+    public class TypeWithPropertiesNestingItsGenericTypeArgument<T>
+    {
+        public Tuple<T>? Shallow1 { get; set; }
+        public Tuple<Tuple<T>>? Deep1 { get; set; }
+        public Tuple<Tuple<T>, int>? Deep2 { get; set; }
+        public Tuple<int, Tuple<T>>? Deep3 { get; set; }
+        public Tuple<int, int, Tuple<T>>? Deep4 { get; set; }
+        public Tuple<int, int, Tuple<T, T>>? Deep5 { get; set; }
```

Comment:
Lets  add a bit diversity
```suggestion
        public Tuple<int?, Tuple<T>>? Deep3 { get; set; }
        public Tuple<int, int?, Tuple<T?>>? Deep4 { get; set; }
        public Tuple<int, int, Tuple<T, int>?>? Deep5 { get; set; }
```